app/src/main/java/org/stepic/droid/ui/fragments/CourseDetailFragment.java

```diff
@@ -215,22 +207,10 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
         //VIEW:
         coursePropertyList = new ArrayList<>();
         joinCourseSpinner = new LoadingProgressDialog(getActivity());
-        footer = ((LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate(R.layout.fragment_course_detailed_footer, coursePropertyListView, false);
-        coursePropertyListView.addFooterView(footer);
-        instructorsCarousel = ButterKnife.findById(footer, R.id.instructors_carousel);
-
-        header = ((LayoutInflater) getActivity().getSystemService(Context.LAYOUT_INFLATER_SERVICE)).inflate(R.layout.fragment_course_detailed_header, coursePropertyListView, false);
-        coursePropertyListView.addHeaderView(header);
+        LayoutInflater layoutInflater = LayoutInflater.from(getContext());
```

Comment:
Why not just [`getLayoutInflater()`](https://developer.android.com/reference/android/support/v4/app/Fragment.html#getLayoutInflater())?

Reply:
right